lessons/8_loops/index.html

```diff
@@ -0,0 +1,308 @@
+<html>
+  <head>
+    <title>Javascript orientation session!</title>
+    <link rel="stylesheet" href="../../reveal.js/css/reveal.css" />
+    <link rel="stylesheet" href="../../reveal.js/css/theme/moon.css" />
+    <link rel="stylesheet" href="../../reveal.js/lib/css/zenburn.css" />
+    <style>
+      .reveal pre {
+        margin: 0;
+      }
+
+      .monospace {
+        font-family: monospace !important;
+      }
+
+      .illustration {
+        border: 0px;
+      }
+
+      .reveal li {
+        margin-bottom: 0.5em;
+      }
+
+      .reveal pre {
+        width: 100%;
+      }
+
+      .reveal pre code {
+        cursor: pointer;
+        max-height: 600px;
+      }
+
+      .reveal li pre {
+        width: fit-content;
+      }
+
+      .logo {
+        position: fixed;
+        /* Sit on top of the page content */
+        z-index: 2;
+        /* Specify a stack order in case you're using a different order for other elements */
+      }
+    </style>
+  </head>
+
+  <body>
+    <a href="https://www.redi-school.org/"
+      ><img src="../../redi-school-logo.jpg" class="logo"
+    /></a>
+    <div class="reveal">
+      <div class="slides">
+        <section data-markdown>
+          <textarea data-template>
+            # Lecture 8
+            ### Loops
+            </textarea
+          >
+        </section>
+
+        <section>
+          <ul>
+            <h4>
+              Last week we learned about
+              <strong>Arrays</strong>
+              and
+              <strong>Objects</strong>
+            </h4>
+          </ul>
+          <pre class="fragment"><code data-trim class="hljs">
+                const items = ["Item 01", "Item 02", "Item 03", "Item 04"];
```

Comment:
Maybe something more close to reality) like teachers, favorite books, colors...?

Reply:
Updated